content/docs/rtl-support/index.md

```diff
@@ -0,0 +1,114 @@
+# RTL support
+The goal is to have full RTL support for all of ionic.
+If you don't need RTL support, you can set `$include-rtl: false`, and your css bundle will be smaller.
```

Comment:
Fair enough. Maybe we should have a "standard performance tips" article where this is included "If you don't need RTL..."